source/definitions.rst

```diff
@@ -136,19 +131,6 @@ Definitions
        Site identity.
        Used in order to refer to a “logical” identity of a site.
 
-       At the STA, the following formats can be used:
```

Comment:
We should add what format is allowed as site id. For instance, Site id is a string, e.g. KK+AG0503=001DL001 or DL1